public/app/plugins/datasource/loki/modifyQuery.ts

```diff
@@ -162,6 +163,7 @@ export function addLabelToQuery(
   const parserPositions = getParserPositions(query);
   const labelFilterPositions = getLabelFilterPositions(query);
   const hasStreamSelectorMatchers = getMatcherInStreamPositions(query);
+  const labelFormatPositions = getNodePositionsFromQuery(query, [LabelFormatExpr]);
```

Comment:
@matyax could we add a comment explaining why are we adding it after the LabelFormat?

Reply:
Good idea!

Reply:
Done. Let me know if you have a better suggestion